components/src/core/drawer/drawer.component.scss

```diff
@@ -12,6 +16,7 @@ $transition: width 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
     border-right: 0;
     &.pxb-drawer-side-border {
         border-left: 1px solid rgba(0, 0, 0, 0.12);
```

Comment:
why is this a border-left?

Reply:
These are the RTL styles.   The border-right style is down below, line 35.